app/models/storage_tables/attachment.rb

```diff
@@ -14,5 +14,9 @@ class Attachment < ApplicationRecord
     def download
       association(:blob).klass.service.download("#{blob_key}#{checksum}==")
     end
+
+    def path
+      association(:blob).klass.service.path_for(checksum)
```

Comment:
Hoeft de blob_key hier niet bij in?

Reply:
Klopt inderdaad, had eerder een spec erin die slaagde met verkeerde path. Nu een test toegevoegd die path checked of document ook daadwerkelijk op die locatie is.